iac/uni-nav.deploy.yml

```diff
@@ -0,0 +1,113 @@
+Description: UniNav hosting on S3 and CloudFront 
+
+Parameters:
+
+    Stage:
+        Type: String
+        Description: The Environment/Stage for which to create the stack
+
+    Domain:
+        Type: String
+        Description: UniNav Hosting Domain
+        AllowedValues:
+            - topcoder.com
+            - topcoder-dev.com
+
+Conditions: 
+    IsProd: !Equals [ !Ref Domain, topcoder.com ]
+    IsNotFeature: 
+        !Or [!Equals [ !Ref Stage, dev ], Condition: IsProd]
+
+Mappings:
+
+    DomainMap:
+
+        topcoder.com: 
+            CertArn: arn:aws:acm:us-east-1:409275337247:certificate/8cabe19d-6559-41c6-8dfb-66f8080d27d4
+            HostedZoneId: Z1B7U7QS3AAI1X
+            RoleArn: arn:aws:iam::409275337247:role/learning-paths-v5-container-role
+
+        topcoder-dev.com: 
+            CertArn: arn:aws:acm:us-east-1:811668436784:certificate/8e56f24a-1eaf-4289-8c21-db2c0761ecf4
+            HostedZoneId: Z2CIRG3R0ZSGFQ
+            RoleArn: arn:aws:iam::811668436784:role/learning-paths-v5-container-role
+
+Resources:
+
+    Store:
+        Type: AWS::S3::Bucket
+        DeletionPolicy: Delete
+        Properties:
+            AccessControl: PublicRead
```

Comment:
We need to find a solution for hosting the file in S3 that doesn't require the bucket to be open for Public Read. Can you look into best practices for hosting web assets that solve that, possibly using CloudFront and an access origin restriction?